docs/design/add-in-commands.md

```diff
@@ -1,38 +1,69 @@
 ---
 title: Basic concepts for add-in commands
```

Comment:
Can we change solitary "add-ins" throughout these topics to "Office Add-ins" in the title, H1, and first paragraph? This will help with SEO since the ranking keyword for our docs is indeed "Office Add-ins". Alternatively, we can change to "add-ins for Word, Excel, PowerPoint, and Outlook."  Or both. Not just "add-ins" because it's a wasted SEO opportunities.

Reply:
I hear the concern. The issue is the feature is literally called "add-in commands".